rosmip_joint_publisher:
  type: joint_state_controller/JointStateController
  publish_rate: 50

rosmip_imu_controller:
  type: imu_sensor_controller/ImuSensorController
  publish_rate: 50

rosmip_balance_controller:
    type: rosmip_control/RosMipController
    kp_theta: 0.8
    kd_theta: 0.15

